samplers/jaegerremote/sampler_remote_options.go

```diff
@@ -37,6 +40,36 @@ type config struct {
 	logger                  logr.Logger
 }
 
+func getEnvOverrideOptions() []Option {
+	var options []Option
+	if os.Getenv("OTEL_TRACES_SAMPLER") == "jaeger_remote" {
+		args := strings.Split(os.Getenv("OTEL_TRACES_SAMPLER_ARG"), ",")
+		for _, arg := range args {
```

Comment:
Hi @yurishkuro , does it work to ignore errors here or should we report it to the user?